file/file-alfresco/src/main/java/uk/gov/justice/services/file/alfresco/requester/AlfrescoFileRequester.java

```diff
@@ -29,6 +29,10 @@
     String alfrescoWorkspacePath;
 
     @Inject
+    @GlobalValue(key = "alfrescoPdfContentWorkspacePath", defaultValue = "/service/api/node/workspace/SpacesStore/")
```

Comment:
Initially thought to return the original file if the "transformPdf" param is false, I agree to your point, I will change the Alfresco implementation then I will incorporate the changes here.